examples/node/inscribe.js

```diff
@@ -1,79 +1,55 @@
-import { Ordit } from "@sadoprotocol/ordit-sdk"; //import Ordit
-import { base64_encode } from "./utils";
+import { Ordit } from "@sadoprotocol/ordit-sdk"
 
-const WORDS = "<MNEMONIC WORDS>"; // Generated HD wallet seed phrase
-const IMG_BASE64 = base64_encode("./azuki-small-compressed.png"); // relative path to image
+const MNEMONIC = "<MNEMONIC>"
 
 async function main() {
-  // Load wallet
+  // init wallet
   const wallet = new Ordit({
-    bip39: WORDS,
-    network: "mainnet"
+    bip39: MNEMONIC,
+    network: "testnet"
```

Comment:
Can we use the Network type from `packages/sdk/src/config/types.ts` instead of magic strings?

Reply:
This is a js file. 👀

Reply:
Oh it's an example. Nevermind then.